middleware.go

```diff
@@ -64,22 +51,22 @@ func LogEndpoint(fields ...LogOption) endpoint.Middleware {
 	return func(e endpoint.Endpoint) endpoint.Endpoint {
 		return func(ctx context.Context, request interface{}) (response interface{}, err error) {
 			if opts[LogRequest] {
-				LogMessage(ctx, fmt.Sprintf("request: %+v", request))
+				_ = LogMessage(ctx, fmt.Sprintf("request: %+v", request))
```

Comment:
Qu'est ce qu'il se passe si un LogMessage renvoit une erreur.
Dans quel cas cela peut se produire ?
Est-ce que ça ne va pas nous empêcher de voir qu'on a un problème avec notre système de logging et donc notre capacité à détecter une erreur sur le système ?

Reply:
As you cannot log the error return by your logger... I don't see what you can do.

Reply:
How about falling back to the classic "log" in case of an error so at least you can figure out that there is a problem.